scripts/ci/pre_commit/pre_commit_json_schema.py

```diff
@@ -30,7 +30,7 @@
 from jsonschema.validators import extend, validator_for
 
 if __name__ != "__main__":
-    raise Exception(
+    raise RuntimeError(
```

Comment:
I kind of wonder if all these blocks with the `if __name__ != "__main__"` pattern should just `sys.exit`, but that’s probably a separate discussion. This is OK.

Reply:
We could replace it by `SystemExit`, there is no problem to do it now.